cove_ofds/templates/cove_ofds/explore.html

```diff
@@ -268,21 +268,25 @@ <h4 class="panel-title">
     </div>
 
     <div class="row">
-      <div class="panel {% if can_download_geojson%}panel-success{% else %}panel-danger{% endif %}">
+      <div class="panel {% if any_nodes_with_geometry == False and any_spans_with_geometry == False %}panel-warning{% elif can_download_geojson == False %}panel-danger{% else %}panel-success{% endif %}">
           <div id="visualisation-panel" class="panel-heading pointer" role="region" aria-expanded="true"
                aria-controls="validationTable" data-toggle="collapse" data-target="#mapVisualisation">
               <h4 class="panel-title">
                   <span class="glyphicon glyphicon-collapse-up"></span>{% trans 'Visualisation' %}
               </h4>
           </div>
           <div id="mapVisualisation" class="collapse in panel-body">
-            {% if can_download_geojson %}
-            <p>{% trans 'The GeoJSON version of your data is visualised on the map below. You should check that nodes and spans appear in the correct location. If not, you may need to ' %}<a href="https://open-fibre-data-standard.readthedocs.io/en/0.1-dev/guidance/publication.html#how-to-transform-coordinates-to-the-correct-coordinate-reference-system">{% trans 'transform your coordinates to the correct coordinate reference system' %}</a>.</p>            
-              <div id="map" class="map" style="width: 100%; height: 600px;"></div>
-            {% else %}
+            {% if any_nodes_with_geometry == False and any_spans_with_geometry == False %}
+              <div>
+                <p>{% trans 'Your data cannot be visualised on a map because it does not contain coordinates. If you expected your data to include coordinates, you should check your mapping and data pipeline for errors. For more information, see ' %}<a href="https://open-fibre-data-standard.readthedocs.io/en/0.1-dev/reference/schema.html#geometry">{% trans 'Geometry' %}</a>.</p>
+              </div>
+            {% elif can_download_geojson == False %}
               <div>
                 <p>{% trans 'Your data cannot be visualised because it is not available in GeoJSON format. For more information, see ' %}<a href="#download-panel">{% trans 'data conversion' %}</a>.</p>
               </div>
+            {% else %}
+              <p>{% trans 'The GeoJSON version of your data is visualised on the map below. You should check that nodes and spans appear in the correct location. If not, you may need to ' %}<a href="https://open-fibre-data-standard.readthedocs.io/en/0.1-dev/guidance/publication.html#how-to-transform-coordinates-to-the-correct-coordinate-reference-system">{% trans 'transform your coordinates to the correct coordinate reference system' %}</a>.</p>            
```

Comment:
Noting I'm not sure this way of doing trans tags will work for good translations, but as we don't have I18N at the moment that is fine - all the trans tags will need reviewing anway

Reply:
feel free to leave trans tags off the messages for now

Reply:
I followed the existing use of `trans` tags in this file, but happy to adjust as required. Do you want them removed?

Reply:
It's fine.